README.md

```diff
@@ -545,6 +579,32 @@ types.py:5: error: Argument 1 to "foo" has incompatible type "int"; expected "st
 Found 2 errors in 1 file (checked 1 source file)
 ```
 
+### TypedDict
+
+As of 3.8, python's `typing` module provides the `TypedDict` class, allowing
+you to specify the member types and key names of a `dict. This is useful for
+declaring data classes as a distinct type, while still affording the ease of
+use of a conventional dict. From the docs:
+
+> TypedDict declares a dictionary type that expects all of its instances to
+> have a certain set of keys, where each key is associated with a value of a
+> consistent type. This expectation is not checked at runtime but is only
+> enforced by type checkers.
+
+Here's how to use it:
+
+```python3
+>>> import typing
+>>> class MyData(typing.TypedDict):
+...     a: int
+...     b: int
+...
+>>> MyData(a=1, b=2)
+{'a': 1, 'b': 2}
+```
+
+See more [in the docs](https://docs.python.org/3.10/library/typing.html#typing.TypedDict).
```

Comment:
rest of resources link to the docs first in the section in bullet points. lets stay consistent

Reply:
sure, makes sense.